doc/rfcs/002-update-buffer-titles-in-guest.md

```diff
@@ -0,0 +1,57 @@
+# Update Buffer Titles in Guest's Workspace
+
+### Status
+
+Proposed
+
+### Summary
+
+We want to add the functionality to have title changes of text-buffers from the host using Teletype to be reflected in the guests' buffers.
+
+### Motivation
+
+As described in [#147](https://github.com/atom/teletype/issues/147), when the host renames a file, the guest should see those changes reflected in their workspace as well.
+
+### Explanation
+
+There is a new file in _Teletype_, called _buffer-file.js_:
+  This File acts as a filler for the `File` object. It implements:
```

Comment:
Note, although I've submitted [PR 293](atom/text-buffer/pull/293), we do not need `existSync`, since `buffer.isModified` is overridden by editor-binding's monkey edits to `false`.